pages/documentation/simulation/sensitivity/openlf.md

```diff
@@ -76,41 +76,160 @@ We only support for the moment balance type `PROPORTIONAL_TO_GENERATION_P_MAX` a
 
 The contingency management consists in calculating the sensitivity values for post-contingency states of the network. A post-contingency state of the network is the state of the network after an outage (most of the time, the loss of a line). In the particular case of DC flows approximation, the post-contingency sensitivity values can be computed using the pre-contingency sensitivity values and some flow transfer factors. Thus, the same LU decomposition is used both for the pre-contingency analysis and for the post-contingency analysis.
 
-#### Loss of a single branch
-
-The most frequent contingency is the loss of a single branch.
+#### N-1 case
+Let us start with the most classic case, where a single branch was lost by the network.
  
-Let's introduce $$s_{b,ij,mk}$$ the sensitivity of an increase of 1 MW at bus $$b$$ on branch $$(i,j)$$ where the branch $$(m,k)$$ represents the outage.
+Let $$s_{b,ij,mk}$$ be the sensitivity of an increase of 1 MW at bus $$b$$ on branch $$(i,j)$$ where the branch $$(m,k)$$ has been disconnected from the network.
 We want to compute this sensitivity.
 
-We call $$b^1$$ the right-hand side vector corresponding with an increase of 1 MW at bus $$b$$. We call $$\theta^1$$ the state vector of voltage angles obtained by solving the equation system on the pre-contingency network that has as right-hand side $$b^1$$.
+Let $$b^1$$ be the right-hand side vector corresponding with an increase of 1 MW at bus $$b$$.
+
+Let $$b^2$$ be the right-hand side vector corresponding with an increase of 1 MW at bus $$m$$ plus a decrease of 1 MW at bus $$k$$.
 
-We call $$b^2$$ be the right-hand side vector corresponding with an increase of 1 MW at bus $$m$$ and a decrease of 1 MW at bus $$k$$. We call $$\theta^2$$ the state vector of voltage angles obtained by solving the equation system on the pre-contingency network that has as right-hand side $$b^2$$.
+Let $$\theta^1$$ be the vector of voltage angles obtained solving the network contraints system on the pre-contingency network with right-hand side $$b^1$$.
 
-Note that both $$\theta^1$$ and $$\theta^2$$ are built using the same LU decomposition of the constraints matrix $$J$$.
+Let $$\theta^2$$ be the vector of voltage angles obtained solving the network contraints system on the pre-contingency network with right-hand side $$b^2$$.
+Notice that $$\theta^1$$ and $$\theta^2$$ are built using the same LU decomposition of the constraints matrix $$J$$.
 
 Let $$s_{b,ij}$$ be the sensitivity of an increase of 1 MW at bus $$b$$ on branch $$(i,j)$$ on the pre-contingency network, we recall that:
 
 $$
 s_{b,ij} = \frac{\theta^1_i-\theta^1_j}{X_{i,j}}
 $$
 
-Let $$s_{mk,ij}$$ be the sensitivity of an increase of 1 MW at bus $$m$$ and a decrease of 1 MW at bus $$k$$, on the pre-contingency network.
+Let $$s_{mk,ij}$$ be the sensitivity of an increase of 1 MW at bus $$m$$ plus a decrease of 1 MW at bus $$k$$, on the pre-contingency network.
```

Comment:
Can you here add an index to bus `b`?

Reply:
I chosed to not add an index to bus b here because s_{mk,ij} is not relative to bus b. The same factor can be used both to compute sensitivity from b to (i,j) in case (m,k) is disconected and from another bus b' to (i,j) with (m,k) disconnected.

Let me know if you still want to add an index here.